mergecounts.wdl

```diff
@@ -48,5 +68,6 @@ task MergeCounts {
 
     runtime {
         memory: 4 + (2*length(inputFiles))
+        docker: "biowdl/mergecounts:latest"
```

Comment:
latest is unstable. Please use git tag or something to tag this release as 1.0 (or 0.1 whatever you like) and do the same with the docker tag.